pulsar-broker/src/test/java/com/yahoo/pulsar/client/api/PartitionedProducerConsumerTest.java

```diff
@@ -59,7 +59,9 @@ public void testRoundRobinProducer() throws Exception {
         log.info("-- Starting {} test --", methodName);
 
         int numPartitions = 4;
-        DestinationName dn = DestinationName.get("persistent://my-property/use/my-ns/my-partitionedtopic1");
+        final String specialCharacter = "! * ' ( ) ; : @ & = + $ , /\\ ? % # [ ]";
```

Comment:
Instead of changing the topic name on an existing test, can you add a new test that just does the minimum (lookup and connection) with the special characters name?